static/fonts.less

```diff
@@ -1,62 +0,0 @@
-@font-folder: "fonts";
```

Comment:
This file has been moved to KaTeX/katex-fonts.  See https://github.com/KaTeX/katex-fonts/pull/10.